src/detect-engine-alert.c

```diff
@@ -172,19 +172,34 @@ static inline void RuleActionToFlow(const uint8_t action, Flow *f)
     }
 }
 
+/** \internal
+ */
+static inline PacketAlert PacketAlertSet(
+        DetectEngineThreadCtx *det_ctx, const Signature *s, uint64_t tx_id, uint8_t alert_flags)
+{
+    PacketAlert pa;
```

Comment:
if we init `pa` as `PacketAlert pa = { s->num, s->action, s, ... }` it would give a compile error if a new field was added that isn't set here. Would that make sense to avoid future issues?